docs/concepts.rst

```diff
@@ -33,3 +33,36 @@ There are two types of tables:
            :language: python
            :start-after: [START temp_table_example]
            :end-before: [END temp_table_example]
+
+.. _load_file_working:
+
+
+How load_file Works
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. figure:: /images/defaultPath.png
```

Comment:
Is this on Lucid chart ?

Reply:
Worth sharing with the team-mates so that when it needs to be updated, we can just update it in Lucid Chart and generate the image

Reply:
Yes, it's lucid charts, I made the diagram shareable and added a link to the document.